### PR TITLE
Add extra help for Explainer link(s)

### PR DESCRIPTION
**Pull request by dlaliberte**
This text is mostly from the Chrome launch process.

@jyasskin 
@cwilso

### CODE REVIEW

**Comment by dlaliberte** [comment]
Here is what it looks like initially:

image

The last line about being a Googler needs to be changed, but this made me wonder whether the mentorship program might be turned into a public service, with mentors outside of Google.

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:313]: Is there any way this text can link to the explainer_links section instead of just naming it?

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:371]: Spell check:
```suggestion
        You will need to publish this explainer by sharing it on a public forum,
```

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:373]: Link to the instructions for making a WICG proposal: https://github.com/WICG/proposals#what-does-a-proposal-look-like

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:376]: Link to the identities of the WICG co-chairs: https://wicg.github.io/admin/charter.html#chairs.

Comment (jyasskin) [inline client-src/elements/form-field-specs.js:376]: This sentence doesn't really belong as help for the Explainer field, since people aren't going to be trying to move to the WICG at the same time they're first writing their explainer.

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:379]: This probably belongs in the help text for the Start prototyping phase.

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:382]: This also probably belongs in instructions for the Start Prototyping phase, rather than for the initial explainer, which should be more about having a clear problem statement and engaging with the community.

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:387]: Not just if you're a Googler. Maybe:
```suggestion
        If you want help, ask for a <a href="https://sites.google.com/a/chromium.org/dev/blink/spec-mentors">specification mentor</a>.
```

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:384]: This should be the first sentence in this section.

**Review by jyasskin — CHANGES_REQUESTED** [review]

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [reply client-src/elements/form-field-specs.js:387]: (As you [noticed](https://github.com/GoogleChrome/chromium-dashboard/pull/2566#issuecomment-1343442404). The service already gets requests from non-Googlers.)

File: client-src/elements/form-field-specs.js
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:382]: Moving to Intent to Prototype

File: client-src/elements/form-field-specs.js
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:384]: done

File: client-src/elements/form-field-specs.js
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:387]: Sounds good.

File: client-src/elements/form-field-specs.js
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:313]: I believe so.

Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:313]: But it doesn't help much since it doesn't seem to autoscroll.  Maybe something else on the page is overriding the default autoscroll to the anchor.

**Comment by dlaliberte** [comment]
Not sure I got all the edits to reflect your comments, but here is the current state:

image

In the Start Prototyping stage, I added extra help to the Spec link field.  Not sure this is the best way to do it, but it looks like lots more restructuring of these fields might be helpful.

image

File: client-src/elements/form-field-specs.js
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js]: done

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:408]: I think this doesn't really fit as extra_help for the spec field, since it happens long before most teams should actually start writing the specification. We might need to just drop this text for this PR, but @jrobbins, what do you suggest for help text that belongs on the phase rather than any particular field?

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:381]: Since the "Initial Public Proposal" field sits before the "Explainer" field in the guide, maybe this should talk instead about "If you've already made an initial public proposal, post your explainer to that thread. Otherwise, your initial proposal can refer to you explainer." And then put the link to making a WICG proposal in the extra_help for the "Initial Public Proposal" field.

File: client-src/elements/form-field-specs.js
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:383]: done

File: client-src/elements/form-field-specs.js
Comment (jrobbins) [reply client-src/elements/form-field-specs.js:408]: I agree that this is too high-level for field-specific extra-help.

We are moving toward a more approval-centric approach with more approval gates owned by more teams, even if some of the gates are auto-approved when the feature owner fills in all fields.  As part of that UX change, there is a new location available for process help text.  We call it the "questionnaire", but it can be a paragraph of any text, not neccessarily questions.   Is there some team that would logically be the ones giving the advice proposed in this part of the PR? Adoption, Interop, Security, Privacy, UX, A11y, Testing, Debuggability, DevRel, Enterprise, or overall API owners?

Alternatively, I suppose that we could add support for extra help on the stages themselves.

Comment (jyasskin) [reply client-src/elements/form-field-specs.js:408]: I think this is an "overall API owners" thing, or at least, none of the other areas in your list seem right. The core text comes from https://www.chromium.org/blink/launching-features/#step-1-incubating-write-up-use-cases-and-scenarios-in-an-explainer, which is from the API owners.

Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:408]: I'll leave this extra_help out for now.  I like the idea of the help text (and maybe extra_help as well) for the stages.  

While each type of approver should perhaps be able to get a view of the data that is specifically only what they are interested in, the users who are making changes to the data regarding a particular stage might be interested in everything that needs to be done for that stage.  If those users should not have to look under half a dozen other views (for each approval process) to find all the info they need.

**Comment by dlaliberte** [comment]
Latest version:

image

**Comment by dlaliberte** [comment]
> Latest version:

The extra_help for the Initial public proposal URL is probably not what you wanted.  But I'm not sure exactly what you wanted.

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:360]: Since WICG doesn't use discourse anymore, how about:

```suggestion
        Link to the first public proposal to create this feature.`,
    extra_help: html`
        If there isn't another obvious place to propose your feature, create a
        <a href="https://github.com/WICG/proposals#what-does-a-proposal-look-like">WICG proposal</a>.
        You can use your proposal document to help you socialize the problem with other vendors
        and developers.`
```

I'm not certain we need that last sentence; it's not necessarily connected to having the public proposal document.

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:388]: Nit:
```suggestion

```

File: client-src/elements/form-field-specs.js
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:386]: I think this doesn't really belong here. We ought to have a field that captures the plans around moving to a CG, and this help text would be useful there, but until that field exists, I'm inclined to leave this paragraph out.
```suggestion
```

**Review by jyasskin — APPROVED** [review]
With changes like the ones below (or overriding direction from Jason), I'm happy with this.

File: client-src/elements/form-field-specs.js
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:382]: done

File: client-src/elements/form-field-specs.js
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:360]: done.

File: client-src/elements/form-field-specs.js
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:386]: done

**Comment by dlaliberte** [comment]
Latest version:

image

**Review by jrobbins — APPROVED** [review]

**Pull request merged**
